app/screens/settings/display_settings/display_settings.js

```diff
@@ -20,6 +20,7 @@ export default class DisplaySettings extends PureComponent {
     static propTypes = {
         navigator: PropTypes.object.isRequired,
         theme: PropTypes.object.isRequired,
+        enableTheme: PropTypes.bool.isRequired,
```

Comment:
Why this? themes are always enabled

Reply:
If only one theme is available, does it make sense to allow them to switch. Should this be view-only if only one theme is available?

Reply:
I would propose hiding the "Theme" setting option in Display settings if the theme cannot be changed.